frontend/packages/dev-console/integration-tests/features/Knative/channel-ODC-3225.feature

```diff
@@ -0,0 +1,148 @@
+    And user clicks on the Edit Application Groupings
+    And user will click on the Application dropdown on the modal
+    And user selects the Application
+    And user clicks on Save button
+    Then user will see the changed Application Groupings of Channel
+
+
+@regression
+Scenario: Edit Application Groupings to unassigned action on Channel
+    Given user is at Developer Perspective
+    And user is having a Channel inside an applicaiton group on the Topology page
+    When user right clicks on the Channel to open the context menu
+    And user clicks on the Edit Application Groupings
+    And user will click on the Application dropdown on the modal
+    And user selects the unassigned item
+    And user clicks on Save button
+    Then user will see that Channel is without an application group
+
+
+@regression
+Scenario: Edit Labels action on Channel
+    Given user is at Developer Perspective
+    And user is having Channel on the Topology page
+    When user right clicks on the Channel to open the context menu
+    And user clicks on the Edit Labels
+    And user adds the label
+    And user clicks on the Save button on the modal to save labels and close the modal
+    And user right clicks on the Channel to open the context menu
+    And user clicks on the Edit Labels
+    Then user will see the newly added label
+
+
+@regression
+Scenario: Edit Annotations action on Channel
+    Given user is at Developer Perspective
+    And user is having Channel on the Topology page
+    When user right clicks on the Channel to open the context menu
+    And user clicks on the Edit Annotations
+    And user adds the annotations
+    And user clicks on the Save button on the modal to save annotation and close the modal
+    And user right clicks on the Channel to open the context menu
+    And user clicks on the Edit Annotations
+    Then user will see the newly added annotation
+
+
+@regression
+Scenario: Edit Channel action on Channel
+    Given user is at Developer Perspective
+    And user is having Channel on the Topology page
+    When user right clicks on the Channel to open the context menu
+    And user clicks on the Edit InMemoryChannel
+    Then user will see the YAML editor to edit the channel
+
+
+@regression
+Scenario: Delete Channel action on Channel
+    Given user is at Developer Perspective
+    And user is having Channel on the Topology page
+    When user right clicks on the Channel to open the context menu
+    And user clicks on the Delete InMemoryChannel
+    And user clicks on the Delete button on the modal
+    Then Channel will get deleted
+    And user won't be able to see the Channel again
+
+
+@regression
+Scenario: Sidebar for the Channel
+    Given user is at Developer Perspective
+    And user is having Channel on the Topology page
+    When user clicks on the Channel to open the sidebar
+    Then user will see the Resources tab
+    And user will see the Details tab
```

Comment:
It would be good to capture the sections shown for the channels in the details tab.